java/src/jmri/util/EarlyInitializationPreferences.java

```diff
@@ -73,10 +79,32 @@ private void load() {
 
         } catch (IOException ex) {
 //            log.warn("Loading startup preferences from {} failed", FILENAME);
-            preferences.setProperty("sun.java2d.uiScale", "1");
-            store();
+            setupNewPreferences();
         }
     }
 
+    private void setupNewPreferences() {
```

Comment:
Should this routine be Windows-only?  jmri.util.SystemType doesn't start Swing.

Reply:
Is this scaling only a Windows thing? How does Mac and Linux handle HighDPI screens? I wasn't aware of any of this before I started working on #10338 so I don't know.

But the `JMRI_InitPreferences.properties` isn't only about `sun.java2d.uiScale`. It could be used for any system property that either JMRI or the user wants to set in the beginning when JMRI starts. So I think the file should be created if it doesn't exists on all platforms but maybe store `sun.java2d.uiScale` only on Windows?.

Reply:
That sounds right:  just Windows for the sun.java2d.uiScale option